src/main/resources/assets/securitycraft/lang/de_de.json

```
@@ -556,7 +556,7 @@
 	"help.securitycraft.universal_block_reinforcer_lvl3.info": "Der Universelle Blockverstärker verstärkt zahlreiche Vanilla-Blöcke und wandelt sie in ihre SecurityCraft Varianten um. Rechtsklicke den Universellen Blockverstärker, und platziere Blöcke in den oberen Slot um sie zu verstärken, dann schließe das GUI. Um verstärkte Blöcke wieder in ihre Vanilla-Varianten umzuwandeln, nutze den unteren Slot. LvL3 hat unendlich Nutzungen.",
 	"help.securitycraft.briefcase.info": "Der Aktenkoffer ist ein einfaches, Passwort gesichertes Item, welches bis zu zwölf Items aufnehmen kann. Rechtsklicke es nach dem Herstellen, um den vierstelligen Code einzugeben, der den Aktenkoffer öffnet. Ein erneuter Rechtsklick öffnet den Aktenkoffer und du musst den richtigen Code eingeben, um an die Items zu kommen. Um den Code zu entfernen oder den Besitzer zu ändern, lege den Aktenkoffer in deine Zweithand und rechtsklicke mit einem Universellen Passwortwechsler/Universellen Besitzeränderer in deiner Haupthand.",
 	"help.securitycraft.universal_key_changer.info": "Der Universelle Passwortwechsler erlaubt dir, das Passwort deiner Passwort gesicherten Blöcke zu ändern. Rechtsklicke einen Passwort gesicherten Block, um das GUI zu öffnen, dann gib das neue Passwort ein.",
-	"help.securitycraft.secret_sign_item.info": "Das Geheime Schild funktioniert wie das Vanilla-Schild, mit dem Unterschied, dass nur der Besitzer und gewhitelistete Spieler den Text lesen können. Stelle sicher, dass keiner den Block, an dem das Schild ist, zerstören kann!",
+	"help.securitycraft.secret_sign_item.info": "Das Geheime Schild funktioniert wie das Vanilla-Schild, mit dem Unterschied, dass nur der Besitzer und gewhitelistete Spieler den Text lesen und mit ihm interagieren können. Stelle sicher, dass keiner den Block, an dem das Schild ist, zerstören kann!",
```

```suggestion
	"help.securitycraft.secret_sign_item.info": "Das Geheime Schild funktioniert wie das Vanilla-Schild, mit dem Unterschied, dass nur der Besitzer und gewhitelistete Spieler den Text lesen und mit dem Schild interagieren können. Stelle sicher, dass keiner den Block, an dem das Schild ist, zerstören kann!",
```

People aren't interacting with the text, but with the sign. Same with the other lines.